src/ansys/fluent/core/session_solver.py

```diff
@@ -116,9 +121,11 @@ def build_from_fluent_connection(self, fluent_connection):
         self._build_from_fluent_connection(fluent_connection)
 
     @property
-    def svar_data(self) -> SVARData:
-        """Return the SVARData handle."""
-        return service_creator("svar_data").create(self.svar_service, self.svar_info)
+    def solution_variables_data(self) -> SolutionVariablesData:
+        """Return the SolutionVariablesData handle."""
+        return service_creator("solution_variables_data").create(
```

Comment:
These strings `"svar"` and `"svar_data"` passed through `service_creator` are internal and renaming can be avoided (and avoid the corresponding pycosole change).